Src/Particle/AMReX_ParticleUtil.H

```diff
@@ -222,8 +224,11 @@ void enforcePeriodic (P& p,
             // clamp to avoid precision issues;
             if (p.pos(idim) == phi[idim]) p.pos(idim) = plo[idim];
             if (p.pos(idim) > phi[idim]) p.pos(idim) = std::nextafter( (amrex::ParticleReal) phi[idim], (amrex::ParticleReal) plo[idim]);
+            shifted = true;
         }
     }
```

Comment:
The assertion from PeriodicShift in terms of enforce_periodic should be: 
`        AMREX_ASSERT( (p.pos(idim) >= phi[idim] ) and ( p.pos(idim) < plo[idim] ));`

Reply:
Added this back in.